src/components/main/TeamPageController.jsx

```diff
@@ -30,7 +31,10 @@ export default class TeamPageController extends FalcorController {
       if (!this.state.data) {
         return <NotFound />;
       }
-      const teamData = this.state.data.semesters.latest;
+      const teamData = _.map(this.state.data.semesters.latest, team => ({
```

Comment:
Not sure what this change is doing. I don't see you changing how `teamData` is used anywhere. Does this change stem from the rename?

I think as a general rule that would make Find and Replace PR's in general easier to review, it would be nice if all changes that require some sort of logic change and not just the straight up F&R were noted in the PR text.

Reply:
Yeah sorry my bad, I just caught some prop types errors in the browser while testing since when I had done ESlint fixes I had changed some components to (in line with what we were doing with not accepting falcor pseudo-arrarys) to only accept arrays, but hadn't changed all the places that called it apparently. Will try and make that clearer another time!